web/src/message_view.ts

```diff
@@ -767,6 +776,89 @@ export let show = (raw_terms: NarrowTerm[], show_opts: ShowMessageViewOpts): voi
             then_select_offset,
         );
 
+        if (id_info.first_unread_msg_id_pending_server_verification) {
+            const params = message_fetch.get_parameters_for_message_fetch_api({
+                anchor: "first_unread",
+                num_before: 0,
+                num_after: 0,
+                cont() {
+                    // ignore
```

Comment:
This comment should explain more about what is being skipped by passing an empty function here.

Reply:
Added.
```
// Success callback is sufficient to do what we need to do
// here, we don't need another post fetch callback.
````